snooty.toml

```diff
@@ -25,6 +25,7 @@ sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 driver-short = "PyMongo"
 driver-long = "PyMongo, the MongoDB synchronous Python driver,"
 driver-async = "PyMongo Async"
+django-odm = "MongoDB Backend for Django"
```

Comment:
Can this be "Django MongoDB Backend" or are we required to say "MongoDB Backend for Django" ?

Reply:
I can change it! updated